assets/js/_js/default/contact_form.js

```diff
@@ -0,0 +1,17 @@
+function sendForm(alertText) {
```

Comment:
Is this really needed? I see that HTML takes care of the validation of the fields

Reply:
This is needed to show the message alert, if the js validation doesn't exist it will show the alert without checking if you are sending content.

Reply:
Solved here: https://github.com/curacultura/curacultura-website/pull/74#discussion_r1479444236